projects/ngx-d3timeline/src/lib/axis/axis.service.ts

```diff
@@ -17,7 +17,12 @@ export class AxisService {
   resourceAxis$ = this.scalesService.scales$.pipe(
     map(scales =>
       this.createAxis(
-        new ResourceAxisTickRenderer(scales.scaleBand),
+        new ResourceAxisTickRenderer(
```

Comment:
This can be injected into the constructor, see comment below on removing other constructor parameters

Reply:
As above will address in #107